pallets/author-mapping/src/lib.rs

```diff
@@ -237,7 +239,7 @@ pub mod pallet {
 	#[pallet::getter(fn account_and_deposit_of)]
 	/// We maintain a mapping from the AuthorIds used in the consensus layer
 	/// to the AccountIds runtime (including this staking pallet).
-	type MappingWithDeposit<T: Config> = StorageMap<
+	pub type MappingWithDeposit<T: Config> = StorageMap<
```

Comment:
We are only using this in the crate context, we can make it pub crate only

Reply:
That's a valid point, but our other pallets also have them just `pub` (eg https://github.com/PureStake/moonbeam/blob/master/pallets/asset-manager/src/lib.rs#L119). And I worry that making stuff too restricted will make it hard to write precompiles.

Shall we make a followup ticket to go through the repo, make the storage items private, and expose intentional public interfaces?

Reply:
https://purestake.atlassian.net/browse/MOON-933